### PR TITLE
LPS-150420 Audit and refactor tests on WebContent | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-150420

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 1]
```diff
@@ -33,28 +40,26 @@ definition {
 	test AddLocalizedWebContent {
 		property testray.component.names = "Localization";
 
-		task ("Add a Web Content Display widget to page") {
+		task ("Given a Web Content creator adds a new web conent") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:43]: Given a site designer has a page where a Web Content Display widget displays a web content

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 2]
```diff
@@ -33,28 +40,26 @@ definition {
...
+			JSONWebcontent.addWebContent(
+				content = "Web Content Content",
+				groupName = "Test Site Name",
+				title = "Web Content Title");
+		}
+
+		task ("And has a Web Content Display and Language Selector to widget page") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:50]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 3]
```diff
@@ -33,28 +40,26 @@ definition {
...
 
-		task ("Add a web content") {
-			JSONWebcontent.addWebContent(
-				content = "Web Content Content",
-				groupName = "Test Site Name",
-				title = "Web Content Title");
-		}
-
-		task ("Select the web content in Web Content Display") {
+		task ("When the Web Content creator selects the web content in Web Content Display") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:62]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 4]
```diff
@@ -64,13 +69,13 @@ definition {
 			IFrame.closeFrame();
 		}
 
-		task ("View web content is shown in Web Content Display") {
+		task ("Then the Web Content creator can see the web content in Web Content Display") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:72]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 5]
```diff
@@ -64,13 +69,13 @@ definition {
...
 			WebContentDisplayPortlet.viewContent(
 				webContentContent = "Web Content Content",
 				webContentTitle = "Web Content Title");
 		}
 
-		task ("Translate the web content to Chinese") {
+		task ("When the Web Content creator adds a Chinese translation to web content") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:78]: When the site designer adds a Chinese translation to web content via Web Content Display

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 6]
```diff
@@ -84,210 +89,226 @@ definition {
 			PortletEntry.publish();
 		}
 
-		task ("View web content is shwon in Chinese") {
+		task ("And change the site's language to Chinese") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:92]: Then the site designer could view translation after changing the user language to Chinese

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 7]
```diff
@@ -84,210 +89,226 @@ definition {
...
 			LanguagePortlet.selectLanguagePG(
 				locale = "zh-CN",
 				textIcon = "中文-中国");
+		}
 
+		task ("Then the Web Content creator can see the web content in Chinese") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:98]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 8]
```diff
@@ -84,210 +89,226 @@ definition {
...
 			WebContentDisplayPortlet.viewContent(
 				webContentContent = "世界您好 Content",
 				webContentTitle = "世界您好 Title");
 		}
 
-		task ("View web content is shown in English") {
+		task ("When the Web Content creator changes the site's language to default") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:104]: When the site designer changes the user language back to default

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 9]
```diff
@@ -84,210 +89,226 @@ definition {
...
 			LanguagePortlet.tearDownLanguagePG();
+		}
 
+		task ("Then the Web Content creator can see the web content in English") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:108]: Then the site designer could view the web content in English

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind refactoring the task blocks according to the first case? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind refactoring the task blocks according to the first case? Thanks

Hi @Tim-Cao, PR updated. Would you mind have a look at?

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/8258

**Pull request closed without merging**